brain/converter/crypto.py

```diff
@@ -21,7 +21,9 @@ def hashpass(p):
     @salt - a random string of saltlength bytes generated to hash the password
 
     '''
-    salt = base64.b64encode(os.urandom(saltlength))
+    global salt_length
```

Comment:
You've already defined (and shouldn't be global) this on line 13, so `global salt_length` can be removed.